snapcraft/internal/common.py

```diff
@@ -263,3 +263,12 @@ def get_pkg_config_paths(root, arch_triplet):
     ]
 
     return [p for p in paths if os.path.exists(p)]
+
+
+def _get_os_release_info():
```

Comment:
We use the underscore in front of the name to mark something as internal to the file.
This function is called from outside of this file, so please remove the underscore.